NitroxServer/Serialization/World/WorldPersistence.cs

```diff
@@ -188,7 +188,7 @@ private World CreateFreshWorld()
 
             world.SimulationOwnershipData = new SimulationOwnershipData();
             world.PlayerManager = new PlayerManager(players, config);
-            world.EventTriggerer = new EventTriggerer(world.PlayerManager);
+            world.EventTriggerer = new EventTriggerer(world.PlayerManager, gameData);
```

Comment:
All of the managers here should be decoupled from persistence; ie, we shouldn't pass in gameData and have them understand serialization.  Would recommend pulling data off of the event trigger when saving like we do for everything else above (line 33-37).